setup.py

```diff
@@ -8,9 +8,9 @@
 # NOTE: These are tested in `continuous_integration/test_imports.sh` If
 # you modify these, make sure to change the corresponding line there.
 extras_require = {
-    "array": ["numpy >= 1.15.1"],
+    "array": ["numpy >= 1.16"],
```

Comment:
I'd prefer the mins to include the patch unless there is a reason not to. 

```suggestion
    "array": ["numpy >= 1.16.0"],
```

Reply:
See NEP-29: https://numpy.org/neps/nep-0029-deprecation_policy.html#detailed-description
I understand that it states (albeit in a less than perfectly clear wording) that support for patch versions would overdeliver on the contract stated by the NEP. The idea is that bumping a patch version is (typically) fairly painless.

From personal experience on xarray, pint, and my various personal projects, leaving the patch version free relieves quite a bit of pain from the maintainers while being hardly an issue for any user.